docs/blog/2019-09-26-gatsby-perf/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Web Performance 102: Keeping Gatsby Sites Blazing Fast"
+author: Dustin Schau
+date: 2018-09-26
+image: images/rocket.jpg
+showImageInArticle: false
+tags: ["performance", "v2", "lighthouse", "webpagetest"]
+---
+
+Blazing fast isn't just a marketing buzzword. There's real business value in a fast site, and whether or not your users realize it, they _want_ a blazing fast site. Gatsby is my preferred solution for making fast sites and making sites fast. Gatsby balances smart, performant defaults with excellent developer ergonomics, making it my preferred solution for creating sites that my users love. However, blazing fast isn't _just_ a goal that once hit can be lauded and bragged about. Blazing fast is something that needs to be regularly measured, understood, and improved. Each new library that's added, each new screen, each new feature runs the risk of bogging down your site, lessening the meaningful part of blazing in blazing fast. If we're _merely_ fast, our competitive advantage in our speed is lost! In this post, I'll deep dive on this notion of blazing fast: how do we measure blazing fast? How did we improve upon Gatsby v1's gold standard to squeeze even more performance out of every site? What are some simple, meaningful metrics to _know_ if your site is blazing fast? Let's get to it.
+
+> At the BBC, 10% of users are lost for every additional second the site takes to load.
+
+## Measuring Speed
+
+It can oftentimes be overwhelming and exhausting to truly _know_ all the performance optimizations and [web performance acronyms][web-perf] that are helpful to make your site blazing fast. H/2, PRPL, RAIL, FLIP, SPA, SW, TTI, oh my! Gatsby internalizes many of these practices so that you get _for free_ a blazing fast site with smart defaults out of the box. It's valuable to show rather than tell, so let's go over some performance monitoring tools and techniques so that you can learn how to measure the speed of your site and measure your site's performance. In particular, I'll go over two tools: Lighthouse and Webpagetest, and two metrics, TTI and Speed Index. Armed with the knowledge of these tangible metrics and tools you can be empowered yourself to _know_ whether your site is actually blazing fast or merely an imposter.
```

Comment:
Need to actually mention/discuss Speed Index more, and I'll bring these home in the Webpagetest with a simple table/graph comparing Gatsby v1 to Gatsby v2.